program_transformers/src/lib.rs

```diff
@@ -28,6 +28,23 @@ pub mod error;
 mod token;
 mod token_metadata;
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct AccountInfo {
+    pub slot: u64,
+    pub pubkey: Pubkey,
+    pub owner: Pubkey,
+    pub data: Vec<u8>,
+}
```

Comment:
Should we keep `write_version` in here from the original serialized types?  Maybe this is a better discussion for https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/issues/177 but I have been suspecting we might want to utilize it at some point in the future.

Background on what that value is: https://docs.rs/solana-geyser-plugin-interface/1.18.5/solana_geyser_plugin_interface/geyser_plugin_interface/struct.ReplicaAccountInfoV2.html#structfield.write_version

It might help us fix something that could be affecting us in account-based updates.

Reply:
Depending on the commitment level that we want to support, `write_version` is only important for processed. I'd recommend staying only on finalized, support processed + confirmed is a huge pain from our experience in the steamboat (custom gPA indices).

Reply:
ok, yeah I did not know it only would be used for processed and confirmed.  I assumed if there were two different account updates in the same slot in different transactions, it would be used even at finalized.  But if not, we might as well leave it out.